Azure-Samples/chat-with-your-data-solution-accelerator
# IV changes for Explore Data & Delete Data page

## Purpose

* Required by #321 
* This PR updates Explore & Delete pages on the Admin website corresponding to the Integrated Vectorization changes.
* Decouples the Explore & Delete page from the Search functionality.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
If 'AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION' is True, the Explore & Delete page should show the corresponding Ingested documents. 
No changes in the UI.
```
